File: quan.lua/src/test/config/item/ItemIds.lua
local ItemConfig = require("test.config.item.ItemConfig")

---
---道具ID
---@author 自动生成
local ItemIds = {}

---
---道具1
---@return int
function ItemIds.item1()
    return ItemConfig.getByKey("item1").id
end

---
---道具2
---@return int
function ItemIds.item2()
    return ItemConfig.getByKey("item2").id
end

return ItemIds

File: quan.cs/Test/Config/Item/ItemIds.cs
namespace Test.Config.Item
{
    /// <summary>
    /// 道具ID<br/>
    /// 自动生成
    /// </summary>
    public class ItemIds 
    {
        /// <summary>
        /// 道具1
        /// </summary>
        public static int Item1 => ItemConfig.GetByKey("item1").Id;

        /// <summary>
        /// 道具2
        /// </summary>
        public static int Item2 => ItemConfig.GetByKey("item2").Id;
    }
}
